google_api_availability_android/android/settings.gradle

```
@@ -18,7 +18,7 @@ pluginManagement {
 
 plugins {
     id "dev.flutter.flutter-plugin-loader" version "1.0.0"
-    id "com.android.application" version "8.1.0" apply false
+    id "com.android.application" version "8.7.0" apply false
 }
 
  include ":app"
```

Missing new line indicating the end of file:
```suggestion
 include ":app"
 
```